src/builder/create_interaction_response.rs

```diff
@@ -278,6 +278,26 @@ impl CreateInteractionResponseMessage {
 /// They're same according to Discord, see
 /// [Autocomplete docs](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-autocomplete).
 pub type AutocompleteChoice = CommandOptionChoice;
+impl AutocompleteChoice {
+    pub fn new(name: impl Into<String>, value: impl Into<serde_json::Value>) -> Self {
```

Comment:
```suggestion
    pub fn new(name: impl Into<String>, value: impl Into<Value>) -> Self {
```

It's already imported via `internal::prelude::*`